### PR TITLE
fix: misc bug fixes in creation

### LINKED ISSUE #6634: Package Manager - dueling scrollbars
In package manager, the height of the package list seems to be about the height of the header too tall, which causes both the package list and the overall Composer window to have scroll bars. The double scroll effect is very frustrating.

Fix: Alter the height of the package list control so that two scroll bars are not necessary.

Capture

### LINKED ISSUE #6861: Link to Manage Profiles from Channel Settings is broken
## Describe the bug

- Navigate to settings
- Under Bot Service Channels, select Manage profiles in dropdown
- You should be taken to the new Profile management location on Publish page, but this is not currently working.

image

cc @Dewain27

### LINKED ISSUE #6898: Bot Settings - Manage Connections on the Action Bar not working properly
## Describe the bug

Manage connections on the Bot Settings page does not take you the tab for connections.  It seems that it doesn't do anything after moving to the tabular view of Bot Settings.

image

## Version

Version:  1.4.0-nightly.234518.8e98823
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser

- [X] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [X] Windows
- [ ] Ubuntu

## To Reproduce

1. Open a Bot Project or Create a New one
2. Go to the Bot Settings Page
3. Click Manage Connections (Be on any tab other than the Connections tab)
4. Button does nothing

## Expected behavior

By clicking the Manage Connections button, it should take you to the bot settings page tab for connections

### PR DESCRIPTION
**Pull request by pavolum**
## Description

This PR includes the bug fixes for broken links to the tab view within the settings page, broken links to the 'create provisioning profile' page as well as a CSS bug fix for duel scroll bars in package manager

## Task Item

fixes #6861
fixes #6898 
fixes #6634

### CODE REVIEW

**Review by benbrown — DISMISSED** [review]

File: Composer/packages/client/src/pages/publish/Publish.tsx [hunk 1]
```diff
@@ -426,7 +427,13 @@ const Publish: React.FC<RouteComponentProps<{ projectId: string; targetName?: st
       <Pivot
         selectedKey={activeTab}
         styles={{ root: { marginLeft: 12 } }}
-        onLinkClick={(link) => setActiveTab(link?.props?.itemKey || '')}
+        onLinkClick={(link) => {
+          setActiveTab(link?.props?.itemKey || '');
+          if (link?.props.itemKey) {
+            setActiveTab(link.props.itemKey);
+            navigateTo(`/bot/${projectId}/publish/all/#${link.props.itemKey}`);
```
Comment (benbrown) [inline Composer/packages/client/src/pages/publish/Publish.tsx:434]: Is calling navigateTo the right way to update the URL? this doesn't cause extra navigation to happen?

Comment (pavolum) [reply Composer/packages/client/src/pages/publish/Publish.tsx:434]: It does not lead to a page refresh but does rerender the component

File: extensions/packageManager/src/pages/Library.tsx [hunk 2]
```diff
@@ -603,7 +603,7 @@ const Library: React.FC = () => {
           </Link>
         </p>
       </div>
-      <Stack horizontal verticalFill styles={{ root: { borderTop: '1px solid #CCC' } }}>
+      <Stack horizontal verticalFill styles={{ root: { borderTop: '1px solid #CCC', height: 'calc(100% - 150px)' } }}>
```
Comment (srinaath) [inline extensions/packageManager/src/pages/Library.tsx:606]: @pavolum can we indicate what the 150px is for using a variable?

Comment (pavolum) [reply extensions/packageManager/src/pages/Library.tsx]: Okay I have added better naming for the styling.

**Review by cwhitten — APPROVED** [review]

**Pull request merged**